pkg/controller/cainjector/controller.go

```diff
@@ -157,7 +157,7 @@ func (r *genericInjectReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 		return ctrl.Result{}, err
 	}
 	if caData == nil {
-		log.Info("could not find any ca data in data source for target")
+		log.V(logf.WarnLevel).Info("could not find any ca data in data source for target")
```

Comment:
Is this imporant enough  for a warning? Seems like an info message to me...I don't know how often this is likely to occur?